sysutils/ddrescue/Portfile

```
@@ -23,4 +23,7 @@ checksums       rmd160  f34748e1cbf7be897be7051e162d279fa6f2cdb7 \
                 sha256  ce538ebd26a09f45da67d3ad3f7431932428231ceec7a2d255f716fa231a1063 \
                 size    87001
 
-configure.args  CXX=${configure.cxx}
+configure.args  CXX=\"${configure.cxx}\" \
+                CPPFLAGS=\"${configure.cppflags}\" \
+                CXXFLAGS=\"${configure.cxxflags}\" \
+                LDFLAGS=\"${configure.ldflags}\"
```

I would be surprised if the backslashes were required. You also need to use the right arch flags. Since these changes change what gets built, the revision needs to be increased.
```suggestion
variant universal {}
configure.args  CXX="${configure.cxx}" \
                CPPFLAGS="${configure.cppflags}" \
                CXXFLAGS="${configure.cxxflags} [get_canonical_archflags cxx]" \
                LDFLAGS="${configure.ldflags} [get_canonical_archflags ld]"
```

---

lgtm, let's try it.